### PR TITLE
update einfo test-data

### PR DESCRIPTION
**Pull request by bernt-matthias**
another tool that currently fails CI. seems that there is a new db. 

An alternative would be to `assert_text` using one of the major dbs. Also wondering if this new DB needs to be reflected in some of the other tools. But I have not much knowledge of these tools / DBs.

Updated the requirement since it appeared to me that `einfo --db ..`  returns invalid xml, maybe someone could have a look. I guess `decode()` needs to be added also at other places.

FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [x] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/ncbi_entrez_eutils/macros.xml [hunk 1]
```diff
@@ -145,7 +145,7 @@ permissions.
   <xml name="requirements">
     <requirements>
       <requirement type="package" version="3">python</requirement>
-      <requirement type="package" version="1.70">biopython</requirement>
+      <requirement type="package" version="1.78">biopython</requirement>
```
Comment (mvdbeek) [inline tools/ncbi_entrez_eutils/macros.xml:148]: It looks like new biopython returns a combination of bytes and strings on some of the Entrez methods.
That's a bit messy, maybe keep it at a lower biopython version for now ?

**Comment by bgruening** [comment]
ping @bernt-matthias

File: tools/ncbi_entrez_eutils/macros.xml (hunk 1 above)
Comment (bernt-matthias) [reply tools/ncbi_entrez_eutils/macros.xml]: done

**Comment by bernt-matthias** [comment]
Ready for review / comments.

**Review by bgruening — APPROVED** [review]

**Comment by bgruening** [comment]
Thanks.

**Pull request merged**